src/utils/accessibility.js

```diff
@@ -0,0 +1,12 @@
+export const visuallyHidden = {
```

Comment:
Since this is a constant, maybe we should use capital snake case like we do for things in `spacing`.